apps/voting/test/voting.js

```diff
@@ -114,14 +119,14 @@ contract('Voting App', ([root, holder1, holder2, holder20, holder29, holder51, n
         })
 
         it('can change required support', async () => {
-            const receipt = await voting.changeSupportRequiredPct(neededSupport.add(1))
+            const receipt = await voting.changeSupportRequiredPct(neededSupport.add(web3.utils.toBN(1)))
```

Comment:
Ah, for these, I think we can "cheat" by using the string representation instead of requiring a BN.